src/bpftrace.cpp

```diff
@@ -1953,4 +1953,37 @@ bool BPFtrace::is_pid_alive(int pid)
   return true;
 }
 
+void BPFtrace::error(std::ostream &out, const location &l, const std::string &m)
+{
+  // Bison and the semantic analyser stop at the first error they see,
```

Comment:
Not quite true - the semantic analyser reports all the errors it finds in a single pass. What would you consider caching here though?